mvcc/kvstore_test.go

```diff
@@ -405,10 +405,17 @@ func TestStoreRestore(t *testing.T) {
 	if g := b.tx.Action(); !reflect.DeepEqual(g, wact) {
 		t.Errorf("tx actions = %+v, want %+v", g, wact)
 	}
+
+	gens := []generation{
+		generation{created: revision{4, 0}, ver: 2, revs: []revision{revision{3, 0}, revision{5, 0}}},
+		generation{created: revision{0, 0}, ver: 0, revs: nil},
+	}
+	key1 := []byte{0, 0, 0, 0, 0, 0, 0, 3, 95, 0, 0, 0, 0, 0, 0, 0, 0}
```

Comment:
@xiang90 As you see here the key is not matching with []byte("foo"), but coming as byte value []byte{0, 0, 0, 0, 0, 0, 0, 3, 95, 0, 0, 0, 0, 0, 0, 0, 0} which comes as some junk character. All the other data is coming as expected except for the key which is coming as junk value. Not able to figure out the reason for this. Request for your support.
Thank you!